### PR TITLE
Frontend init, Routing

### PR DESCRIPTION
**Pull request by nikiljos**
#### Tracker Topics covered
- [x] ReactJS app initialization

#### Extra Topics covered
- [x] React Router

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "frontend",
```
Comment (aditya-190) [inline frontend/package.json]: Please update the name to your project name.

**Comment by aditya-190** [comment]
please remove the default logo svg for vite -> https://github.com/kalviumcommunity/openbadgx/blob/2ac2c0d70dfb359216953171a85c154c333fa267/frontend/public/vite.svg

**Comment by aditya-190** [comment]
Proceed with merging after the requested changes are done.

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Proceed with merging after the requested changes are done.

**Comment by nikiljos** [comment]
@aditya-190 Pls check now

**Review by aditya-190 — APPROVED** [review]
LGTM

**Review by PatelNeelMahesh — APPROVED** [review]
you can merge pull request.

**Pull request merged**